### DISCUSSION TITLE
[RAC] Button - Any idea why `onPress` is preffered over `onClick` prop?

### DISCUSSION THREAD

**Discussion by thevipinmishra** [adobe/react-spectrum#4935] [Q&A] [answered]
As an Intermediate developer, I've always seen the prop `onClick` in almost all of the UI libraries I've used, including the native button.

But here in React Area Components, I can't even see it listed in the documentation.

The other day, I used a different button to open my RAC Dialog component, inside the DialogTrigegr but that didn't work, and I think the reason was this `onPress` prop.

Can someone please educate me about what's going on with this prop?

**Comment by snowystinger** [comment, marked as answer]
Hey! Thanks for the discussion. I think this blog series will help clear things up for you https://react-spectrum.adobe.com/blog/building-a-button-part-1.html

Let us know if you have further questions!
